# msg para publicar o gait

Vector_leo[]   feet_phases                  # the linear state to reach (x,y,z and derivatives)
Vector_leo[]   contact_status                  # the angular state to reach (roll, pitch, yaw and derivatives)